src/PerformanceSensitiveAnalyzers/UnitTests/ExplicitAllocationAnalyzerTests.cs

```diff
@@ -1,164 +1,152 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.CSharp.PerformanceSensitiveAnalyzers;
+using Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers;
 using Test.Utilities;
 using Xunit;
 using VerifyCS = Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers.UnitTests.CSharpPerformanceCodeFixVerifier<
-    Microsoft.CodeAnalysis.CSharp.PerformanceSensitiveAnalyzers.ExplicitAllocationAnalyzer,
+    Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers.ExplicitAllocationAnalyzer,
     Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
 
 namespace Microsoft.CodeAnalysis.PerformanceSensitive.Analyzers.UnitTests
 {
     public class ExplicitAllocationAnalyzerTests
     {
         [Fact]
-        public async Task ExplicitAllocation_InitializerExpressionSyntax()
+        [WorkItem(7995606, "http://stackoverflow.com/questions/7995606/boxing-occurrence-in-c-sharp")]
+        public async Task Converting_any_value_type_into_interface_reference()
```

Comment:
❓ Can you put the tests in this file back in the original order? The reordering makes it very difficult to review.

Reply:
Done. They were ported in alphabetical order, hence the strange ordering.